.gitlab/lassen-jobs.yml

```diff
@@ -69,7 +69,7 @@ lassen_xl_cuda_tpls:
       - COMPILER_SPEC: xl@16.1.1.10
         CUDA_SPEC: [cuda@10.1.243]
   variables:
-    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double ~int64 +mpi+openmp+cuda~raja+magma+superlu-dist cuda_arch=70 ^superlu-dist+cuda cuda_arch=70 ^magma+cuda cuda_arch=70 ^${CUDA_SPEC}"
+    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double ~int64 +mpi+openmp+cuda~raja+magma~superlu-dist cuda_arch=70 ^superlu-dist+cuda cuda_arch=70 ^magma+cuda cuda_arch=70 ^${CUDA_SPEC}"
```

Comment:
Note to self: This was a change I made in the maintenance branch. I need the superlu team to figure out what is going on. It would be good to have this enabled before the PR is merged.